### DISCUSSION TITLE
Save ruining Bug

### DISCUSSION THREAD

**Discussion by qfmp1012** [niels747/2D-Weather-Sandbox#90] [General]
Anyone knows how to prevent this from happening. it ruined 2 saves and this one is one of them
![Screenshot (8)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/58a15151-026c-42a1-9113-e50bfcda7e43)
![Screenshot (7)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/62a19aa0-72a3-4afb-98b4-6f978f9b0070)

**Comment by Meso22** [comment]
Do you use Google Chrome?

**Reply by qfmp1012** [reply]
Nah I use Microsoft edge

**Reply by Meso22** [reply]
I would reccomend trying out chrome, it might help the issue, I hope so.

**Reply by qfmp1012** [reply]
I ran it on chrome and it still appeared

**Reply by niels747** [reply]
What are your hardware specs? I have only seen something like this before on intel HD graphics.

**Reply by qfmp1012** [reply]
Yea my laptop uses intel

**Reply by qfmp1012** [reply]
alr so i found out i just have to wait for it to go away then it stops after a few seconds
but another thing popped up that only happens when i have over 300 vertical resolution
![Screenshot (9)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/4fcbfd32-3a22-4575-900a-1a0b02aa71e0)

![Screenshot (10)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/2972eddc-d4f2-44c1-b87f-72c0a713ad25)

its not that serious because 300 reso is not bad
